conda-forge.yml

```diff
@@ -1,11 +1,3 @@
-appveyor:
```

Comment:
Why is this all removed?... win+osx do not run here because the recipe is noarch.

Reply:
ah, that makes sense. this was a shot in the dark to try to understand why another package was failing that I'm working on  (https://github.com/conda-forge/staged-recipes/pull/12143) When I updated the spdx here I noticed the win+osx CIs weren't being run and interpreted it to mean that the win+osx packages weren't being updated for some reason.

noarch makes sense! I'll revert to the first commit then.